### PR TITLE
FIAinit Fixes round 1

### PR DESCRIPTION
**Pull request by PoweredByPot**
FIAinit - converted the big "if" to a switch
FIAinitBASES - just a spacing update to match
randomRifle - Spacing and local variable 
dress - took 'teamPlayerDefaultLoadout' out of templates and put it directly in dress, randomized choice

### CODE REVIEW

File: A3-Antistasi/functions/REINF/fn_FIAinit.sqf [hunk 1]
```diff
@@ -17,121 +17,74 @@ if (unlockedVests isEqualTo []) then {removeVest _unit} else {removeVest _unit;
...
-					[_unit,unlockedGrenadeLaunchers] call A3A_fnc_randomRifle;
-					}
-				else
-					{
-					[_unit,unlockedRifles] call A3A_fnc_randomRifle;
-					};
-				}
-			else
-				{
-				if (_typeX != SDKUnarmed) then {[_unit,unlockedRifles] call A3A_fnc_randomRifle};
```
Comment (Spoffy) [inline A3-Antistasi/functions/REINF/fn_FIAinit.sqf]: I think this line is missing? I'm not certain, but I think this needs to be in a 'default' statement.

Comment (PoweredByPot) [reply A3-Antistasi/functions/REINF/fn_FIAinit.sqf]: nope, this was a backwards way of saying "everyone else gets a basic rifle" but I have instead declared a case for each class of rebel troop.

I _could_ still create a default class, and did consider it, but ultimately it should never be reached.

Comment (Spoffy) [reply A3-Antistasi/functions/REINF/fn_FIAinit.sqf]: I'd be happier if there was a default case, just in case something falls through.

Comment (PoweredByPot) [reply A3-Antistasi/functions/REINF/fn_FIAinit.sqf]: done

**Pull request merged**